client/state/selectors/test/get-account-recovery-reset-user-data.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getAccountRecoveryResetUserData } from '../';
+
+describe( 'getAccountRecoveryResetUserData()', () => {
+	it( 'should return the userData substate tree.', () => {
+		const expectedUserData = {
+			user: 'userlogin',
+			firstName: 'Foo',
+			lastName: 'Bar',
+			url: 'site.example.com',
+		};
+		const state = {
```

Comment:
It's obvious from the code that the data isn't mutated, but it might be worth `deepFreeze`ing the state in case of future regressions